app/controllers/application_records_controller.rb

```diff
@@ -22,19 +22,27 @@ def create
   def csv_export
     start_date = parse_american_date(params.require :start_date)
     end_date = parse_american_date(params.require :end_date)
-    params.require :department_ids
-    @records = ApplicationRecord.in_department(params[:department_ids])
-               .between(start_date, end_date)
+    if params[:department_ids]
```

Comment:
I know, I've been brainstorming ways to do it but the only thing I can see to abstract out is ApplicationRecord.in_department. The between(start_date, end_date) thing relies on parameters that are different in all 3 cases. Abstracting the "between" thing will only remove 2 lines from each controller because of how eeo_data works.
Do you have any thoughts?